src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -76,6 +76,13 @@ private async Task<bool> IsEnabledAsync<TContext>(string feature, TContext appCo
 
             if (featureDefinition != null)
             {
+                //
+                // Default to true for requirement type All
+                if (featureDefinition.RequirementType == RequirementType.All)
+                {
+                    enabled = true;
```

Comment:
What do we want the behavior to be when `IgnoreMissingFeatureFilters` is true and no feature filter is found?
Currently, we treat this as a disabled feature since RequirementType is Any. But when RequirementType is All, should that be an enabled feature or disabled?

Reply:
I think it should be disabled. It should have the same effect as this configuration:
```json
"FeatureManagement": {
 "DisabledFeature": {
            "EnabledFor": []
        }
}
```
In that case, the code needs modification to treat a feature with no filters as disabled feature.

Reply:
I was also curious about this. In the current state, it would be evaluated true. 

I think that would be unexpected, but it felt messier to have the AND logic be false- until one filter evaluates to true. Do we know what the use case for using IgnoreMissingFeatureFilters is?

Reply:
Okay! That makes sense, I'll adjust it.